config/default/foia_wizard.settings.yml

```diff
@@ -99,52 +99,52 @@ messages:
     value: "<p>Contact your agency employer for your own personnel or retirement records.</p>\r\n"
     format: rich_text
   m32:
-    value: "<p>If you are a former employee, visit the National Personnel Records Center website for information on submitting a request.</p>\r\n"
+    value: "<p>Visit the <a href=\"https://www.archives.gov/personnel-records-center \">National Personnel Records Center</a> for information on submitting a request.</p>\r\n"
```

Comment:
Looks like there's a space at the end of the `href`. Might cause problems in some browsers. 🤔 I don't know if I checked that in mine. If you search for ` \">` you might find them all.